README.md

```diff
@@ -26,14 +26,17 @@ Please choose the version you like to install. We will use nightly version `mast
 | Version  |  Stable |  Suggested Kubernetes Version |
 |----------|:-------:|------------------------------:|
 |  master  |    N    | v1.23 and above |
-|  v0.2.0  |   Yes   | v1.19 - 1.22 |
+|  v0.3.0  |    Y    | v1.19 - v1.24 |
```

Comment:
@DmitriGekhtman @akanso  We have not run any compatibility test. Do you think it's better to remove it or make it vague？

Reply:
How did we determine compatibility in the past?

Reply:
1. based on kubernetes dependency.  for example, currently we import 1.23 packages
2. the API we are using, check kubernetes API breaking changes.
3. some testing envs and production envs. for example. I used 1.23 for dev. our prod is 1.18
4. experience..

Reply:
Let me use kind to create different clusters and see if v0.3.0 works. I will cover 1.18 and above.

Reply:
Ok, that's much appreciated!

We should figure out if we can automate the compatibility check as part of the release process.
The combinatorial explosion of Ray x K8s x KubeRay is difficult, but 
{latest Ray} x K8s x {release branch Kuberay}
could be straightforward to automate in the future.

cc @wilsonwang371 for thoughts too

Reply:
```
# preparation
 git clone https://github.com/ray-project/kuberay.git
 git checkout v0.3.0
 alias k='kubectl -n ray-system'


# create clusters. 1.18 - 1.24
 minikube start --kubernetes-version=v1.18.0 
 kubectl get nodes


k create -k manifests/cluster-scope-resources/
k apply -k manifests/base
k apply -f ray-operator/config/samples/ray-cluster.complete.yaml

check operator logs and cluster connectivity

minikube stop && minikube delete
```

1.18 reports error in controller because of the ingress version issue. We use networking/v1 instead ext/v1beta so operator can not find the API. But lucky thing is 1.18 still work if people do not enable Ingress. (ingress error in controller can be ignored)
```
2022-08-18T17:46:50.151Z	ERROR	controller-runtime.source	if kind is a CRD, it should be installed before calling Start	{"kind": "Ingress.networking.k8s.io", "error": "no matches for kind \"Ingress\" in version \"networking.k8s.io/v1\""}

```